inc/com/centreon/engine/events/defines.hh

```diff
@@ -42,6 +42,7 @@
 #define EVENT_EXPIRE_COMMENT 15  // removes expired comments
 #define EVENT_EXPIRE_HOST_ACK 16     // remove expired host acknowledgement
 #define EVENT_EXPIRE_SERVICE_ACK 17  // remove expired service acknowledgement
+#define EVENT_ENGINERPC_CHECK 18 // EngineRPC command check
```

Comment:
Just to be sure this define is needed by timed_event to handle grpc command ?

Reply:
It is the way used to force to empty stacks when we have some time. So yes, this is really needed.